client/my-sites/themes/sheet.jsx

```diff
@@ -114,6 +116,24 @@ const ThemeSheet = React.createClass( {
 		);
 	},
 
+	renderFeatures() {
+		const themeFeatures = this.props.taxonomies && this.props.taxonomies.features instanceof Array
+		? this.props.taxonomies.features.map( function( item, i ) {
+			return ( <li key={ 'theme-features-item-' + i++ }><span>{ item.name }</span></li> );
+		} ) : [];
+
+		return (
+			<div>
+				<SectionHeader label={ i18n.translate( 'Features' ) } />
+				<Card className="themes__sheet-features">
+					<ul className="theme-features-list">
```

Comment:
`theme-features-list` doesn't quite conform to the css class naming guidelines:
https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md

(Also may not need the `themes__sheet-features` class at all)